app/src/components/features/rules/RulesListContainer/RulesTable/index.js

```diff
@@ -920,7 +920,7 @@ const RulesTable = ({
         if (record.objectType && record.objectType === "group") {
           return null;
         }
-        const uid = record.createdBy ? record.createdBy : record.lastModifiedBy;
+        const uid = record.createdBy ?? null;
```

Comment:
If `createdBy` is not defined then should we show ananymous?

Reply:
yes that is what happens, when the uid here is null/undefined ([src](https://github.com/requestly/requestly/blob/master/app/src/components/common/UserIcon/index.tsx#L13))

The userIcon is shown as anonymous